logging/src/main/java/com/odysseusinc/logging/LoggingEventMessageFactory.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *
+ * Copyright 2018 Odysseus Data Services, inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Anastasiia Klochkova
+ * Created: August 28, 2018
+ *
+ */
+
+package com.odysseusinc.logging;
+
+import com.odysseusinc.logging.event.*;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+public class LoggingEventMessageFactory {
+
+    private final static List<Class> supportedEvents = Arrays.asList(
+            AddDataSourceEvent.class,
+            AddPermissionEvent.class,
+            AddRoleEvent.class,
+            AddUserEvent.class,
+            AssignRoleEvent.class,
+            ChangeDataSourceEvent.class,
+            ChangeRoleEvent.class,
+            DeleteDataSourceEvent.class,
+            DeletePermissionEvent.class,
+            DeleteRoleEvent.class,
+            DeleteUserEvent.class,
+            FailedDbConnectEvent.class,
+            FailedLogoffEvent.class,
+            FailedLogonEvent.class,
+            LockoutStartEvent.class,
+            LockoutStopEvent.class,
+            SuccessLogoffEvent.class,
+            SuccessLogonEvent.class,
+            UnassignRoleEvent.class);
+
+    public String getMessage(LoggingEvent event) {
+
+        String message = "";
+        if (event instanceof AddDataSourceEvent) {
+            message = "Data source id = " + ((AddDataSourceEvent) event).getId() + ", name = " + ((AddDataSourceEvent) event).getName() + " was created";
```

Comment:
My suggestion is to use String.format, that provides to externalize string templates